alonzo/test/lib/Test/Cardano/Ledger/Alonzo/AlonzoEraGen.hs

```diff
@@ -115,7 +115,7 @@ phase2scripts =
 
 -- | A cost model that sets everything as being free
 freeCostModel :: CostModel
-freeCostModel = CostModel $ 0 <$ fromJust defaultCekCostModelParams
+freeCostModel = CostModel $ 0 <$ fromJust defaultCostModelParams
```

Comment:
I don't see this occurring anywhere else in this PR, Does it affect the golden tests? Or is this a separate change?

If it does affect the golden tests, I wonder if the degenerate cost model is going to have blindspots as a golden test meant to catch changes in the serializations. What do you think, @JaredCorduan ?

Reply:
this will go away as soon as I rebase on top of master (see #2322). sorry for the confusion!

Reply:
it's gone now